cli/batch/cli.py

```diff
@@ -0,0 +1,105 @@
+"""
+Provide implementation of the command line interface's batch commands.
+"""
+import asyncio
+import sys
+
+import click
+from remme import Remme
+
+from cli.batch.forms import ListBatchesForm
+from cli.batch.help import (
+    BATCH_HEAD_ARGUMENT_HELP_MESSAGE,
+    BATCH_IDS_ARGUMENT_HELP_MESSAGE,
+    BATCH_LIMIT_ARGUMENT_HELP_MESSAGE,
+    BATCH_REVERSE_ARGUMENT_HELP_MESSAGE,
+    BATCH_START_ARGUMENT_HELP_MESSAGE,
+)
+from cli.batch.service import Batch
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_URL_ARGUMENT_HELP_MESSAGE,
+)
+from cli.utils import (
+    default_node_url,
+    dict_to_pretty_json,
+    print_errors,
+    print_result,
+)
+
+loop = asyncio.get_event_loop()
+
+
+@click.group('batch', chain=True)
+def batch_commands():
+    """
+    Provide commands for working with batches.
+    """
+    pass
+
+
+def _split_ids(ctx, param, value):
+    """
+    Convert string to list. Remove '[', ']' and split by ','.
+    """
+    if value is None:
+        return
+
+    if value[0] != '[' or value[-1] != ']':
+        click.echo(dict_to_pretty_json({
+            'ids': ['Not a valid list.'],
+        }))
+        sys.exit(FAILED_EXIT_FROM_COMMAND_CODE)
+    value = value[1:-1]
+    values = [c.strip() for c in value.split(',')]
+
+    return values
+
+
+@click.option('--ids', type=str, required=False, help=BATCH_IDS_ARGUMENT_HELP_MESSAGE, callback=_split_ids)
+@click.option('--start', type=str, required=False, help=BATCH_START_ARGUMENT_HELP_MESSAGE)
+@click.option('--limit', type=int, required=False, help=BATCH_LIMIT_ARGUMENT_HELP_MESSAGE)
+@click.option('--head', type=str, required=False, help=BATCH_HEAD_ARGUMENT_HELP_MESSAGE)
+@click.option('--reverse', type=str, required=False, help=BATCH_REVERSE_ARGUMENT_HELP_MESSAGE)
+@click.option('--node-url', type=str, required=False, help=NODE_URL_ARGUMENT_HELP_MESSAGE, default=default_node_url())
+@batch_commands.command('get-list')
+def get_batches(ids, start, limit, head, reverse, node_url):
+    """
+    Get a list of batches.
+    """
+    arguments, errors = ListBatchesForm().load({
+        'ids': ids,
+        'start': start,
+        'limit': limit,
+        'head': head,
+        'reverse': reverse,
+        'node_url': node_url,
+    })
+
+    if errors:
+        print_errors(errors)
+        sys.exit(FAILED_EXIT_FROM_COMMAND_CODE)
+
+    ids = arguments.get('ids')
+    start = arguments.get('start')
+    limit = arguments.get('limit')
+    head = arguments.get('head')
+    reverse = arguments.get('reverse')
+    node_url = arguments.get('node_url')
+
+    remme = Remme(network_config={
+        'node_address': str(node_url) + ':8080',
+    })
+
+    query = {
+        'ids': ids,
+        'start': start,
+        'limit': limit,
+        'head': head,
+        'reverse': reverse,
+    }
+
+    batch_service = Batch(service=remme)
```

Comment:
Actualize the CLI as here — https://github.com/Remmeauth/remme-core-cli/blob/develop/cli/account/cli.py#L54

Reply:
Done.